.vib/kong/cypress/cypress/integration/kong_spec.js

```diff
@@ -51,6 +51,14 @@ it('allows adding a new service and route through admin API', () => {
       expect(response.status).to.eq(201);
     });
 
+    cy.request({
+      method: 'GET',
+      url: `${BASE_URL}/services`,
```

Comment:
IIRC, we shouldn't need to add `${BASE_URL}` here, as Cypress already does so by default.

Reply:
There are 2 endpoints here, the proxy and the API. Will investigate why it's needed but if I'm not mistaken, it's necessary for the test to work.

Reply:
BASE_URL is set some lines above to include the port of the API endpoint.